contracts/contracts/vault/VaultCore.sol

```diff
@@ -112,8 +112,9 @@ contract VaultCore is VaultStorage {
      * @dev Mint OUSD for OUSD Meta Strategy
      * @param _amount Amount of the asset being deposited
      *
-     * Notice: can't use nonReentrant modifier since BaseCurveStrategy's deposit
-     * already has that modifier present
+     * Notice: can't use `nonReentrant` modifier since the `mint` function can
+     * be called within the same transaction as `mintForStrategy` and can cause
```

Comment:
Being called multiple times in the same transaction would be okay? The problem is that it is being called from _inside_ an already nonReentrant method by the vault admin to reallocate/deposit/withdraw funds?

Reply:
Thanks, corrected